front/src/applications/editor/tools/rangeEdition/speedSection/SpeedSectionMetadataForm.tsx

```diff
@@ -103,24 +97,22 @@ const SpeedSectionMetadataForm: FC = () => {
         )}
         {tagSpeedLimits.map(({ id, tag, value }, currentIndex) => (
           <div key={id} className="form-group field field-string">
-            <div className="d-flex flex-row align-items-center">
-              <input
-                required
-                className="form-control flex-grow-2 flex-shrink-1 mr-2 px-2"
-                placeholder=""
-                type="text"
-                value={tag}
-                pattern={
-                  // Insert an invalid pattern to force the error appearance when tag is redundant
-                  tagCounts[tag] && tagCounts[tag] > 1 ? `not ${tag}` : undefined
-                }
-                onChange={(e) => {
-                  const newKey = e.target.value;
-                  setTagSpeedLimits((state) =>
-                    state.map((pair, i) => (i === currentIndex ? { ...pair, tag: newKey } : pair))
-                  );
-                }}
-              />
+            <SelectImprovedSNCF
+              options={speedLimitTags}
+              value={tag}
+              onChange={(tagSelected) => {
+                setTagSpeedLimits((state) =>
+                  state.map((pair, i) =>
+                    i === currentIndex ? { ...pair, tag: tagSelected } : pair
+                  )
+                );
+              }}
+              withSearch
+              withNewValueInput
+              addButtonTitle={t('Editor.tools.range-edition.add-new-tag')}
+              bgWhite
+            />
+            <div className="d-flex flex-row align-items-center my-1">
```

Comment:
By moving this div here, it now looks like that : 

![image](https://github.com/OpenRailAssociation/osrd/assets/113104137/f4c36d05-2f27-4e3f-9f72-ffa031d0f8ba)

But before it was like that, on one line : 

![image](https://github.com/OpenRailAssociation/osrd/assets/113104137/9c7ebc66-a6ca-4d83-8986-c9cc94737953)

To keep this UI, you could : 
- move this div before the select
- add `...attrs` at the end of SelectImproved props (after l.45)
- change the SelectImprovedSNCF props (l.47) like that : `SelectProps<T> & Omit<React.HTMLAttributes<HTMLDivElement>, 'value' | 'onChange'>)` so that we can add more html attributes with props
- add `{...attrs}` to the first div props in the jsx (l.133)
- pass `className="flex-grow-1 mr-2"` in props to SelectImporvedSNCF in SpeedSectionMetadataForm

Reply:
ok but if we try to pass 'inline' in props of SelectImprovedSNCF the classnames are overwritten.
I put a div around the component